vllm/v1/attention/backends/mla/flashmla.py

```diff
@@ -41,6 +42,10 @@ def get_builder_cls() -> type["FlashMLAMetadataBuilder"]:
     def get_impl_cls() -> type["FlashMLAImpl"]:
         return FlashMLAImpl
 
+    @staticmethod
+    def get_supported_block_size() -> list[Union[int, MultipleOf]]:
+        return [64]
```

Comment:
If I understand correctly, this means that the *kernel* block size that is supported is exactly 64? Should we call the function `get_supported_kernel_block_size`?